src/main/java/com/feelmycode/parabole/service/ProductService.java

```diff
@@ -0,0 +1,49 @@
+package com.feelmycode.parabole.service;
+
+import com.feelmycode.parabole.domain.Product;
+import com.feelmycode.parabole.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.data.domain.Pageable;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    @Autowired
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
```

Comment:
`@RequiredArgsConstructor`를 사용하면 자동으로 생성되서 Autowired를 사용하지 않아서 좋을 거 같아요!
```suggestion
@Service
@RequiredArgsConstructor
public class ProductService {

    private final ProductRepository productRepository;

```